README.md

```diff
@@ -50,6 +50,8 @@ export function Example() {
 | `speed`    | Animation speed                                                            | NO       | `number`               | 1       |
 | `spacing`  | Spacing between repeting elements                                          | NO       | `number`               | 0       |
 | `style`    | View style to be applied to Marquee container.                             | NO       | `StyleProp<ViewStyle>` |         |
+| `direction`| Direction in which the repeating elements should scroll                      | NO       | `"up" \| "left" \| "down" \| "right" `  |  `"right"`   |
```

Comment:
The API seem to be too verbose. What do you think about having two props to define the direction (as FlatList is doing), something like:

```typescript
horizontal: boolean
reversed: boolean
```

In this way, in the future, it will be really easy to support `ltr` and `rtl` layouts